wifi_enable.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+echo "This command must be run as SuperUser and using Bash"
+echo "Please give this Pi a unique ID (for the WiFi Access Point):"
+Pass_Phrase=pi2733hotspot
+read Pi_ID
+while true; do
+  echo "Enable Access Point [Y/n]"
+  read enable_ap
+  if [[ $enable_ap = "" || $enable_ap = "y" ]]; then
+    echo "Enabling Access Point..."
+    enable_ap=y
```

Comment:
Should `enable_ap=y` be changed to `enable_ap="y"`?

Reply:
They both work exactly the same in this case